administration/src/mui-modules/theme.ts

```diff
@@ -1,203 +1,197 @@
+    typography: {
+      fontFamily: 'Inter, Roboto, sans-serif',
+      h1: {
+        fontSize: '3.125rem',
+        lineHeight: 1.0,
+        fontWeight: 600,
+        letterSpacing: '-2px',
+        marginBottom: '0.7em',
+      },
+      h2: {
+        fontSize: '2.625rem',
+        lineHeight: 1.2,
+        fontWeight: 600,
+        letterSpacing: '-1px',
+        marginBottom: '0.7em',
+      },
+      h3: {
+        fontSize: '2.125rem',
+        lineHeight: 1.1,
+        fontWeight: 600,
+        marginBottom: '0.7em',
+      },
+      h4: {
+        fontSize: '1.75rem',
+        lineHeight: 1.1,
+        fontWeight: 600,
+        marginBottom: '0.7em',
+      },
+      h5: {
+        fontSize: '1.375rem',
+        lineHeight: 1.1,
+        fontWeight: 600,
+        marginBottom: '0.7em',
+      },
+      h6: {
+        fontSize: '1.125rem',
+        lineHeight: 1.2,
+        fontWeight: 600,
+        marginBottom: '0.7em',
+      },
+      subtitle1: {
+        fontSize: '1.125rem',
+        lineHeight: 1.3,
+        fontWeight: 400,
+        marginBottom: '0.7em',
+      },
+      subtitle2: {
+        fontSize: '1rem',
+        lineHeight: 1.3,
+        fontWeight: 500,
+        marginBottom: '0.7em',
+      },
+      body1: {
+        fontSize: '1rem',
+        lineHeight: 1.3,
+        fontWeight: 400,
+      },
+      body2: {
+        fontSize: '0.875rem',
+        lineHeight: 1.4,
+        fontWeight: 400,
+      },
+      body2bold: {
+        fontSize: '0.875rem',
+        lineHeight: 1.4,
+        fontWeight: 700,
+      },
+      button: {
+        fontSize: '0.875rem',
+        fontWeight: 500,
+        lineHeight: 1.8,
+        textTransform: 'none',
+        textWrap: 'nowrap',
+      },
+      caption: {
+        fontSize: '0.75rem',
+        lineHeight: 1.3,
+        letterSpacing: '0.5px',
+        fontWeight: 400,
+      },
+      overline: {
+        fontSize: '0.625rem',
+        lineHeight: 1.3,
+        fontWeight: 400,
+        textTransform: 'uppercase',
+        letterSpacing: '2px',
+      },
     },
-  },
-  components: {
-    MuiButton: {
-      defaultProps: {
-        variant: 'outlined',
-        color: 'inherit',
-      },
-      styleOverrides: {
-        textInherit: {
-          color: '#5C6065',
-        },
-        outlinedInherit: {
-          borderColor: '#EEEEEE',
-          borderWidth: '2px',
-          color: '#5C6065',
+    components: {
+      MuiButton: {
+        defaultProps: {
+          variant: 'outlined',
+          color: 'inherit',
         },
-        containedInherit: {
-          backgroundColor: '#F6F7F9',
-          borderStyle: 'solid',
-          borderColor: '#EEEEEE',
-          borderWidth: '1px',
-          color: '#5C6065',
-          ':hover': {
-            backgroundColor: '#e9eaedff',
+        styleOverrides: {
+          root: {
+            '&.MuiButton-contained': {
+              '&[href]:hover': {
+                color: '#EEEEEE',
```

Comment:
`import { grey } from '@mui/material/colors'`

```suggestion
                color: grey[200],
```

https://mui.com/material-ui/customization/color/#color-palette